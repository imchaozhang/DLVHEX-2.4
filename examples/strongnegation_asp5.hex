one(a). one(b). two(c). two(d).
in(X,Y) v -in(X,Y) :- one(X), two(Y).
in(b,c).
-in(a,c).
