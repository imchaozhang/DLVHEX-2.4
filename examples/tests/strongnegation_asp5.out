{-in(a,d),-in(b,d)}
{in(a,d),-in(b,d)}
{-in(a,d),in(b,d)}
{in(a,d),in(b,d)}
